pages/order.vue

```diff
@@ -30,47 +30,10 @@
             </template>
             <template v-else>
               <div class="headline">You've selected the {{plan.name}} package</div>
-              <v-simple-table>
-                <template v-slot:default>
-                  <thead>
-                  <tr>
-                    <th class="text-left">Number users</th>
-                    <th class="text-left">Paid monthly</th>
-                    <th class="text-left">Paid yearly</th>
-                  </tr>
-                  </thead>
-                  <tbody>
-                  <tr v-for="range in usersRangeKeys" :key="range">
-                    <td><v-icon>{{getIcon(range)}}</v-icon> {{ range }}</td>
-                    <td>
-                      {{ plan['month'].prices[range] | currency }}/user
-                      (save {{calculateSavings('month', range)}}%)
-                    </td>
-                    <td>
-                      {{ plan['year'].prices[range] | currency }}/user
-                      (save {{calculateSavings('year', range)}}%)
-                    </td>
-                  </tr>
-                  </tbody>
-                </template>
-              </v-simple-table>
-              <NumberInput label="Please select the number of seats:" :min="1" :value="seats" @input="setSeats" />
+              <OrderPricesTable :data="orderPricesTableData" />
+              <NumberInput label="Please select the number of seats:" width="265" :min="1" :value="seats" @input="setSeats" />
```

Comment:
Shortening this input is a big improvement.  Good work seeing and correcting that.